### PR TITLE
Feature/billPc/carlos

### PR DESCRIPTION
**Pull request by carlosmc23**
Please check my code

### CODE REVIEW

File: src/main/java/org/awt05/BillPc.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                System.out.println("The pokemon was added successfully");
+            } else {
+                System.out.println("The pokemon was not deposited,There is not available space");
+            }
+        } else {
+            System.out.println("Unregistered trainer, register please");
+        }
+    }
+
+    public Pokemon withDrawPokemon(int trainerId, int pokemonID) {
```
Comment (ArielYanarico) [inline src/main/java/org/awt05/BillPc.java]: withdraw is a word so your method should name "withdrawPokemon"

File: src/main/java/org/awt05/BoxList.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import java.util.List;
+
+
+public class BoxList {
+
+    private List<PcBox> boxes;
+    private int defaultBox;
+
+    public BoxList() {
+        boxes = new ArrayList<>();
```
Comment (ArielYanarico) [inline src/main/java/org/awt05/BoxList.java]: you could pass limit as parameter in array lists for example:

`boxes = new ArrayList<>(2);`

File: src/main/java/org/awt05/PcBox.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    private int searchPokemon(int idPokemon) {
+        int pokePosition = -1;
+        for (int i = 0; i < pokemonGang.length; i++) {
+            if (pokemonGang[i] != null && pokemonGang[i].getId() == idPokemon) {
+                pokePosition = i;
+                break;
+            }
+        }
+        if (pokePosition < 0) {
+            throw new IllegalStateException("The pokemon has not been found");
```
Comment (ArielYanarico) [inline src/main/java/org/awt05/PcBox.java]: You could use a custom Exception in this case, please sync with Diego and Lucero for doing it

File: src/main/java/org/awt05/billPc/ConsoleBill.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+import java.util.Scanner;
+
+public class ConsoleBill {
+
+    private Scanner scanner;
+    private BillPc bill;
+
+    public ConsoleBill(BillPc bill) {
+        this.bill = bill;
```
Comment (ArielYanarico) [inline src/main/java/org/awt05/billPc/ConsoleBill.java]: rename this attribute to billPC

File: src/test/java/org/awt05/BillPcTest.java [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        BillPc currentBillPc = new BillPc();
+        currentBillPc.registerTrainer(8);
+        currentBillPc.depositPokemon(8, pokemonTest1);
+        currentBillPc.switchBox(8);
+        currentBillPc.depositPokemon(8, pokemonTest2);
+        Pokemon currentPokemon = currentBillPc.withDrawPokemon(8, 46);
+        //when
+        assertEquals(expectedPokemon.getId(), currentPokemon.getId());
+    }*/
+}
```
Comment (carlos8882) [inline src/test/java/org/awt05/BillPcTest.java]: If the test are not necessary, please remove those test

File: src/test/java/org/awt05/PcBoxTest.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        Pokemon pokemonTest1= new Pokemon("pikachu", false, 34);
+        Pokemon[] expectedBox = new Pokemon[24];
+        expectedBox[0] = pokemonTest1;
+        //when
+        PcBox currentBox = new PcBox();
+        currentBox.addPokemon(pokemonTest1);
+
+        //then
+        assertArrayEquals(expectedBox, currentBox.getBox());
+    }*/
```
Comment (carlos8882) [inline src/test/java/org/awt05/PcBoxTest.java]: If the test is not necessary, please remove the test

**Pull request merged**